apps/studio/src/pages/sign-in/singpass/index.tsx

```diff
@@ -1,30 +1,58 @@
+import { useEffect, useState } from "react"
+import NextLink from "next/link"
 import { useRouter } from "next/router"
-import { Flex, Grid, GridItem, Text, useDisclosure } from "@chakra-ui/react"
-import { Button, RestrictedGovtMasthead } from "@opengovsg/design-system-react"
+import {
+  Box,
+  Flex,
+  Grid,
+  GridItem,
+  Link,
+  Text,
+  useDisclosure,
+} from "@chakra-ui/react"
+import {
+  Button,
+  Infobox,
+  RestrictedGovtMasthead,
+} from "@opengovsg/design-system-react"
 import { BiChevronLeft } from "react-icons/bi"
 
 import type { NextPageWithLayout } from "~/lib/types"
 import { PublicPageWrapper } from "~/components/AuthWrappers"
 import { FullscreenSpinner } from "~/components/FullscreenSpinner"
 import { IsomerLogo } from "~/components/Svg"
+import { ISOMER_SUPPORT_LINK } from "~/constants/misc"
 import { SingpassLoginButton } from "~/features/sign-in/components"
 import { UnableToUseSignpassModal } from "~/features/sign-in/components/UnableToUseSignpassModal"
 import { SIGN_IN } from "~/lib/routes"
 import { trpc } from "~/utils/trpc"
 
 const SingpassSignInPage: NextPageWithLayout = () => {
+  const [isSingpassError, setIsSingpassError] = useState<boolean>()
```

Comment:
question - do we need a `useState` and `useEffect` or can we replace it with somehting like this?
```ts
const isSingpassError = Boolean(router.query.error)
```

Reply:
Updated in a116f4da5ed9d10d8d78ae1f8bab668ac4f67b04.